gp-includes/assets-loader.php

```diff
@@ -1,84 +1,94 @@
+function gp_styles_default() {
 	$url = gp_plugin_url( 'assets/css' );
 
-	$styles->add( 'base', $url . '/style.css', array(), '20150717' );
+	// Register our base style.
+	wp_register_style( 'gp-base', $url . '/style.css', array(), '20150717' );
 }
 
-add_action( 'wp_default_styles', 'gp_styles_default' );
+add_action( 'init', 'gp_styles_default' );
 
 /**
- * Register the GlotPress scripts
- *
- * @param WP_Scripts $scripts
+ * Register the GlotPress scripts.
  */
-function gp_scripts_default( &$scripts ) {
+function gp_register_scripts() {
 	$url = gp_plugin_url( 'assets/js' );
 
-	$scripts->add( 'tablesorter', $url . '/jquery.tablesorter.min.js', array( 'jquery' ), '1.10.4' );
-
-	$scripts->add( 'gp-common', $url . '/common.js', array( 'jquery' ), '20150430' );
-	$scripts->add( 'gp-editor', $url . '/editor.js', array( 'gp-common', 'jquery-ui-tooltip' ), '20160329' );
-	$scripts->add( 'gp-glossary', $url . '/glossary.js', array( 'gp-common' ), '20160329' );
-	$scripts->add( 'gp-translations-page', $url . '/translations-page.js', array( 'gp-common' ), '20150430' );
-	$scripts->add( 'mass-create-sets-page', $url . '/mass-create-sets-page.js', array( 'gp-common' ), '20150430' );
+	// Register our standard scripts.
+	wp_register_script( 'tablesorter', $url . '/jquery.tablesorter.min.js', array( 'jquery' ), '1.10.4' );
+	wp_register_script( 'gp-common', $url . '/common.js', array( 'jquery' ), '20150430' );
+	wp_register_script( 'gp-editor', $url . '/editor.js', array( 'gp-common', 'jquery-ui-tooltip' ), '20160329' );
+	wp_register_script( 'gp-glossary', $url . '/glossary.js', array( 'gp-editor' ), '20160329' );
+	wp_register_script( 'gp-translations-page', $url . '/translations-page.js', array( 'gp-editor' ), '20150430' );
+	wp_register_script( 'gp-mass-create-sets-page', $url . '/mass-create-sets-page.js', array( 'gp-editor' ), '20150430' );
 }
 
-add_action( 'wp_default_scripts', 'gp_scripts_default' );
+add_action( 'init', 'gp_register_scripts' );
 
 /**
- * Here we abstract WordPress core's enqueuing functions because...
- * 1. We don't want to print scripts and styles that are meant for the WordPress theme
- * 2. GlotPress enqueues scripts and styles from its template files and if we do that
- *    with wp_enqueue_script() and wp_enqueue_style() WordPress complains that
- *    those functions should only be called inside of wp_enqueue_scripts()
+ * Enqueue one or more styles.
+ *
+ * @param string|array $handles A single style handle to enqueue or an array or style handles to enqueue.
  */
-
-function gp_enqueue_scripts() {
-	global $gp_enqueued_styles, $gp_enqueued_scripts;
-
-	if ( ! empty( $gp_enqueued_scripts ) ) {
-		foreach ( $gp_enqueued_scripts as $handle ) {
-			wp_enqueue_script( $handle );
-		}
+function gp_enqueue_style( $handles ) {
+	// Check to see if $handles is an array, if not, then we can make it one to simplify the next loop.
+	if ( ! is_array( $handles ) ) {
+		$handles = array( $handles );
```

Comment:
This can be simplified to  `$handles = (array) $handles;` or by adding `(array)` to the loop

`foreach ( (array) $handles as $handle ) {`